core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1009,7 +1017,7 @@ static function ($allowedTag)
 
 						if ($objFile !== null)
 						{
-							$arrCache[$strTag] = System::urlEncode($objFile->path);
+							$arrCache[$strTag] = System::urlEncode(System::getContainer()->get('assets.context')->getBasePath() . '/' . $objFile->path);
```

Comment:
```suggestion
							$arrCache[$strTag] = System::urlEncode(System::getContainer()->get('contao.assets.files_context')->getStaticUrl() . $objFile->path);
```

Reply:
Changed in 5a2b802fa3571b4bdd9bb5fcb38dc70950fe7a78.